targets/TARGET_STM/mbed_rtx.h

```diff
@@ -69,6 +69,7 @@
 #define INITIAL_SP              (0x20014000UL)
 
 #elif (defined(TARGET_STM32F401RE) ||\
+       defined(TARGET_STM32F401VE) ||\
```

Comment:
I agree that this patch could be avoided if we change:
defined(TARGET_STM32F401xE)